lib/active_admin/reform/resource.rb

```diff
@@ -0,0 +1,23 @@
+require 'active_support/concern'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.